pom.xml

```diff
@@ -261,8 +267,8 @@
 
     <properties>
         <!-- Legend -->
-        <legend.pure.version>4.3.1</legend.pure.version>
-        <legend.shared.version>0.22.5</legend.shared.version>
+        <legend.pure.version>4.3.2</legend.pure.version>
```

Comment:
why increment pure/shared version? This is usually done pre-release only.

Reply:
I was getting merge conflicts on this.